backend/src/main/scala/cromwell/backend/Backend.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.backend
+
+import cromwell.backend.model.{WorkflowDescriptor, JobDescriptor}
+
+/**
+  * Defines basic structure and functionality to work with a backend.
+  */
+trait Backend {
```

Comment:
The idea is that a developer who is creating a new backend, implements a new CustomBackenActor extending from BackendActor.
After that he/she just need to override backend interface with specific functionality.
With this you are saying that all new custom backends need to implement that interface.